x-pack/plugins/security_solution/scripts/endpoint/resolver_generator_script.ts

```diff
@@ -43,19 +45,52 @@ async function deleteIndices(indices: string[], client: Client) {
   }
 }
 
+interface RoleInfo {
+  role: string;
+}
+
+async function addRole(kbnClient: KbnClient, role: RoleInterface): Promise<RoleInfo | undefined> {
+  if (!role) {
+    return;
+  }
+  const path = `/api/security/role/${role.roleName}?createOnly=true`;
+
+  // add role if doesn't exist already
+  try {
+    console.log(`Adding ${role.roleName} role`);
+
+    const addedRole = (
+      (await kbnClient.request<Promise<Role>>({
+        method: 'PUT',
+        path,
+        body: role,
+      })) as AxiosResponse
```

Comment:
Why is this casting needed?

Reply:
Apparently this api doesn't return any data, so taking this out.